code/__DEFINES/is_helpers.dm

```diff
@@ -255,7 +255,7 @@ GLOBAL_LIST_INIT(glass_sheet_types, typecacheof(list(
 
 #define isblobmonster(O) (istype(O, /mob/living/simple_animal/hostile/blob))
 
-#define isshuttleturf(T) (length(T.baseturfs) && (/turf/baseturf_skipover/shuttle in T.baseturfs))
+#define isshuttleturf(T) (!isnull(T.depth_to_find_baseturf(/turf/baseturf_skipover/shuttle)))
```

Comment:
this is a hot case, don't feel great about the proc call. guess we'll see

Reply:
It might be prudent to make `isshuttleturf` itself a proc for the sake of seeing how expensive it is